content/articles/how-to-add-pure-css-wave-animations-to-your-website/index.md

```diff
@@ -0,0 +1,254 @@
+
+### Introduction
+Adding animations can take you to the next level by making your website seem livelier and more attractive to your users. developers use animations to develop websites that attract more users, increase a websites presence and improve conversions. The downfall of adding animations to a website using JavaScript, as this is the widely used language in adding dynamicity to a website, is animations reduce the performance of a website, especially when the developer fails to select an optimized library. However, there is a turnaround to such a problem, and that is using CSS and HTML only. 
```

Comment:
This is too long. Breakdown it down into paragraphs. Additionally, introduction is only meant to help readers to get general overview of what to expect. Make it as simple as possible!